core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -126,9 +126,13 @@ class KafkaRaftManager[T](
         case spec: InetAddressSpec => {
           netChannel.updateEndpoint(voterAddressEntry.getKey, spec)
         }
+        case _: UnknownAddressSpec => {
```

Comment:
nit: the braces in these `case` matches are not needed

Reply:
Ah makes sense. Removed.